themes/digital.gov/src/scss/new/_card-event.scss

```diff
@@ -86,20 +181,92 @@
 // Upcoming events
 // ==========================================================================
 .card-event-promo {
-  grid-template-areas:
-  "c-date c-content"
-  "c-meta c-meta" !important;
-  grid-template-columns: 1fr 5fr !important;
-
-  .card__media {
-    @include u-border-right("1px", "indigo-warm-80", "solid");
-    grid-area: c-date !important;
-  }
 }
 
-//
 // Past events
 // ==========================================================================
 .card-event-past {
-  border-top: none;
+  @include u-border-top(0);
+  .card__date {
+    @include u-display("block");
+    @include u-border-left(0);
+    @include u-padding-left(0);
+    @include at-media("tablet") {
+      @include u-border-left("1px", "indigo-warm-80", "solid");
+      @include u-padding-left(2);
+    }
+    &-past {
+      @include u-padding-left(0);
+      @include u-text("gray-cool-50");
+      @include u-text(600);
+      transition: color 0.25s;
+    }
+    &-desktop {
+      display: none;
+      @include at-media("tablet") {
+        display: grid;
+        grid-template-areas: "d-month" "d-day" "d-description" !important;
+        grid-template-rows: auto;
+      }
+      .card__date-month {
+      }
+    }
+    &-label {
+      @include u-display("block");
+      @include u-padding(0);
+      @include u-bg("white");
+      @include u-radius("sm");
+      @include u-border("2px", "gray-cool-50", "solid");
+      @include u-font("sans", "sm");
+      @include u-text("gray-cool-50");
+      @include u-text("center");
+      width: 80%;
+      justify-self: center;
+      transition: border 0.25s, color 0.25s;
+    }
+    &-mobile {
+      grid-area: d-description;
+      @include u-display("flex");
+      @include u-flex-direction("row");
+      justify-content: center;
+      @include u-padding(0);
+      @include u-radius("sm");
+      @include u-bg("white");
+      @include u-text("gray-cool-50");
+      @include u-text("center");
+      @include u-font("sans", "sm");
+      @include u-border("2px", "gray-cool-50", "solid");
+      transition: background-color 0.25s, color 0.25s;
+      width: 100%;
+      @include at-media("tablet") {
+        @include u-display("none");
+      }
+      .card__date-label {
+        @include u-border(0);
+        @include u-width("auto");
+      }
+      .card__date-month {
+        font-size: inherit;
+        font-weight: inherit;
+        padding-left: 0.5ch;
+        text-transform: none;
+      }
+      .card__date-day {
+        font-size: inherit;
+        font-weight: inherit;
+        text-transform: none;
+        line-height: inherit;
+        @include u-padding-left(0);
+      }
+    }
+  }
+  &:hover {
+    .card__date-past {
+      @include u-text("gray-cool-60");
+    }
+    .card__date-label {
+      @include u-border("2px", "gray-cool-60", "solid");
+      @include u-text("gray-cool-60");
+    }
+  }
 }
```

Comment:
Are all of these styles necessary for past events? Does that mean future events is the default state?

Reply:
Duplicate and unused styles removed in [f4ece54](https://github.com/GSA/digitalgov.gov/pull/6021/commits/f4ece543373aec364248d25f530ae7aab85ff118).